Common issue faced during the setup of the Vitess operator for Kubernetes (#1848)
vitessio/website: content/en/docs/21.0/get-started/operator.md

<b>Solution:</b> Create a new user and add it to your [docker group](https://docs.docker.com/engine/install/linux-postinstall).

> <b>Issue:</b> Port conflicts when running the port-forward script

Reviewer: As we are adding more elements to the `Common Issues and Solutions` section of this page, I think it would be visually more intuitive if we had sub-section for each issue. Right now each issue is highlighted with a bold keyword `Issue:`. I am thinking it could look like this:

<img width="1352" alt="image" src="https://github.com/user-attachments/assets/4c6c829c-6579-4165-babd-22d3772c5574">

Author: I've implemented the changes as discussed. Now, each issue in the Common Issues and Solutions section is organized into its own sub-section, which should make the layout more intuitive and easier to navigate.

> <b>Issue:</b> Port conflicts when running the port-forward script
> After running the command:
> ```sh
> ./pf.sh &
> alias vtctldclient="vtctldclient --server=localhost:15999"
> alias mysql="mysql -h 127.0.0.1 -P 15306 -u user"
> ```

Reviewer 2: I think that we should simply link to that section:
```suggestion
<b>Issue:</b> Port conflicts when [running the port-forward script (pf.sh)](./#setup-port-forward)
```

Author: I have completed the changes as per your suggestions.

Reviewer 2: I don't see this content there:
```
<b>Issue:</b> Port conflicts when [running the port-forward script (pfs.sh)](./#setup-port-forward)
```

The main point was that we don't need to get into what "running the port-forward" script looks like, we can just link to that section.

> You may encounter an error indicating that Vitess is unable to start because the necessary network ports are already in use by other processes on your system.

Reviewer: Is it an error because the ports that Vitess wants are already busy, or because the ports that port-forwarding (./pf.sh) wants are busy?

The title of the issue describes only port-forwarding but here we mention "Vitess is unable to start", which can be confusing. If we only want to focus on the port-forwarding issue, we should mention that port-forwarding is unable to start.

Author: It was becuase of the ports are busy. Done with the changes as mentioned by you

<b>Solution:</b>
Identify the processes using the ports:

Run the following command to determine which applications or services are occupying the ports Vitess needs:
```sh
#Replace <port_number> with the specific port mentioned in the error message.

> sudo lsof -i :<port_number>

Reviewer 2: This will not only show processes *listening* or bound to that IP address, but also clients that have established a connection to that port. You only want to kill the one listening on it:
```
lsof -I :<port_number> -sTCP:LISTEN
```

Author: Done with the changes as mentioned by you

```

> Stop the conflicting processes:If you find processes using these ports, terminate them using:

Reviewer 2: ```suggestion
If you find processes using these ports, you can terminate them using:
```

kubectl delete -f 101_initial_cluster.yaml
```
Congratulations on completing this exercise!

> [def]: ./#setup-port-forward

Reviewer 2: Can we remove this?
